### PR TITLE
T: Additional tests for move refactoring

### PR DESCRIPTION
**Pull request by dima74**
These tests were originally created when move refactoring was developed, but for some reason haven't been merged together with other code

### CODE REVIEW

File: src/test/kotlin/org/rust/ide/refactoring/move/RsMoveTopLevelItemsTest.kt [hunk 1]
```diff
@@ -897,163 +1048,938 @@ class RsMoveTopLevelItemsTest : RsMoveTopLevelItemsTestBase() {
...
+            struct Bar {}
+
+            fn foo() {
+                let _ = mod1::Bar {};
+            }
+        }
+    """)
+
+    // todo ? рассматривать только `RsPath` которые абсолютные или `this.path.reference().resolve == sourceMod`
+    // todo ? для перемещаемых модулей: если был glob import `use mod1::*`, то добавлять `use mod2::*`
```
Comment (ortem) [inline src/test/kotlin/org/rust/ide/refactoring/move/RsMoveTopLevelItemsTest.kt:1483]: Could be translated into English or even removed? 😄

**Review by ortem — APPROVED** [review]
LGTM except for one comment

**Comment by dima74** [comment]
bors r=ortem

**Comment by dima74** [comment]
bors retry

**Pull request merged**